data/src/main/java/com/microsoft/azure/kusto/data/HttpClientFactory.java

```diff
@@ -47,18 +47,19 @@ static HttpClientFactory getInstance() {
      * @return a new Apache HTTP client
      */
     CloseableHttpClient create(HttpClientProperties providedProperties) {
-        final HttpClientProperties properties = Optional.ofNullable(providedProperties)
+        final HttpClientProperties properties = Optional
+                .ofNullable(providedProperties)
                 .orElse(HttpClientProperties.builder().build());
-        final HttpClientBuilder httpClientBuilder = HttpClientBuilder.create()
+        final HttpClientBuilder httpClientBuilder = HttpClientBuilder
```

Comment:
Can we make it add a new line when indentation decreases following a multi line command